src/filters/filter-panel/filter-panel-results.html

```diff
@@ -24,6 +24,6 @@
         </span>
       </li>
     </ul>
-    <p><a class="clear-filters" ng-click="$ctrl.clearAllFilters()" ng-if="$ctrl.config.appliedFilters.length > 0">Clear All Filters</a></p>
+    <p class="clear-filters"><a href="javascript:void(0);" ng-click="$ctrl.clearAllFilters()" ng-if="$ctrl.config.appliedFilters.length > 0">Clear All Filters</a></p>
```

Comment:
Could this be ``a href=""`` ? The unit test reload error can be fixed by changing the 3 eventFires to just ```.click()```

Reply:
Are you seeing unit test errors?  I am not ??

Reply:
I am getting unit test errors when changing to href=""

Reply:
But I'm fine with how it is currently. Either way using ''''JavaScript:void(0) or a href=""'''